bin/electron-osx-sign-usage.txt

```diff
@@ -66,6 +63,12 @@ DESCRIPTION
   --requirements=requirements
     Specify the criteria that you recommend to be used to evaluate the code signature.
 
+  --restrict
+    Flag to enable restrict mode. Disabled by default. (this will be deprecated soon, see --sign-flags)
+
+  --signature-flags=flags
+    code signature flags. Default to none
```

Comment:
Nit: For the newly added texts, can we have them begin capitalized & end with a period (`.`)? Like "Code signature flags. Default to none.`